port/unix/omrsysinfo.c

```diff
@@ -7354,3 +7376,82 @@ get_Dispatch_IstreamCount(void) {
 	return (uintptr_t)numberOfIStreams;
 }
 #endif /* defined(OMRZTPF) */
+
+uint64_t
+omrsysinfo_get_process_start_time(struct OMRPortLibrary *portLibrary, uintptr_t pid)
+{
+	Trc_PRT_sysinfo_get_process_start_time_enter((unsigned long long)pid);
+	uint64_t processStartTimeInNanoseconds = 0;
+	if (0 != omrsysinfo_process_exists(portLibrary, pid)) {
+#if defined(LINUX)
+		char procDir[OMRPORT_SYSINFO_PROC_DIR_BUFFER_SIZE] = {0};
+		struct stat st;
+		snprintf(procDir, sizeof(procDir), "/proc/%" PRIuPTR, pid);
+		if (stat(procDir, &st) == OMRPORT_SYSINFO_STAT_FAILURE) {
```

Comment:
Constant needs to be on the left. This feedback also applies to other instances where the constant is to the right. https://github.com/eclipse/omr/blob/master/doc/CodingStandard.md#formatting-conditions
```suggestion
		if (OMRPORT_SYSINFO_STAT_FAILURE == stat(procDir, &st)) {
```

Reply:
AIX and zOS don't have a macro for `if (-1 == ret) {`